tox.ini

```diff
@@ -26,7 +26,6 @@ commands =
     benchmark: python -m pytest benchmarks
 
 [testenv:quality]
-basepython=python3.6
```

Comment:
Removing all these versions means we’ll always use Python 3.6.1? I’m not super familiar with how tox chooses its Python version.